app/models/chef.rb

```diff
@@ -1,4 +1,11 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :dish_ingredients, through: :dishes
+   has_many :ingredients, through: :dish_ingredients
```

Comment:
Nice additional association! Because you've set up associations between ingredients and dishes in their controllers, you can also consolidate these 2 lines down to `has_many :ingredients, through: :dishes`